CHANGELOG_PENDING.md

```diff
@@ -4,3 +4,5 @@
 [#542](https://github.com/pulumi/pulumi-java/pull/542)
 
 ### Bug Fixes
+
+- Fix #538, #540: provide compilation failure details
```

Comment:
We might need to put a reminder here which fix goes where, since one of the fixes affects pulumi-language-java language provider, not the SDK, we need to remember to release a new version of that and link it to a new pulumi CLI release.

Reply:
I've added more info